README.md

```diff
@@ -18,5 +18,5 @@ Markdown Source of specifications documents
 ## Some Things to Note
 
  * Much of the site data is in the YAML files in `_data/` (e.g. member institutions, server impls, demos, etc.) make edits there.
- * The latest versions of the APIs are set in `_config.yml`. Change there will get pushed to `.htaccess`, `technical-details.html`, and any other links.
+ * The latest versions of the APIs are set in `_config.yml`. Change there will get pushed to `.htaccess`, `technical-details/`, and any other links.
```

Comment:
Not how to static HTML pages are now directories, e.g. `/technical-details.html` :arrow_right: `/technical-details/index.html`